app/utils/callback/lava.py

```diff
@@ -645,7 +645,7 @@ def add_tests(job_data, job_meta, lab_name, db_options,
             ret_code, plan_doc_id, err = \
                 utils.kci_test.import_and_save_kci_tests(plan, db_options)
             utils.errors.update_errors(errors, err)
-    except (yaml.YAMLError, ValueError) as ex:
+    except (yaml.YAMLError, ValueError, KeyError) as ex:
```

Comment:
This seems to be for different kind of fix, i.e. when anything is missing from the dictionary rather than checking the job status.  It should be in a separate commit.

Reply:
ok, removed from the current PR